docs/index.bs

```diff
@@ -3278,6 +3300,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
                   1. If |navigationPreloadResponse|'s [=response/type=] is "`error`", reject |preloadResponse| with a `TypeError` and terminate these substeps.
                   1. Associate |preloadResponseObject| with |navigationPreloadResponse|.
+                  1. If |timingInfo|'s [=service worker timing info/worker final router source=]is set to {{RouterSourceEnum/"network"}}, Set |timingInfo|'s [=service worker timing info/worker final router source=] be set to {{RouterSourceEnum/"fetch-event"}}.
```

Comment:
Can I ask why this line is needed?  If this is for SW static routing API, I thought the final router source is set in L3284.

Reply:
Sorry for my bad explanation.  I should have said L3285 instead.
This is the end of if-else started from L3236, and we are sure that |source| is fetch-event because we assert.

Reply:
Thanks for the explanation.  Yeah, fetch-event's final router source can be network if fallback.
https://github.com/WICG/service-worker-static-routing-api/blob/main/resource-timing-api.md#correspondence-of-the-matched-source-type-and-the-actual-source-type
I missed that.

However, as we talked offline, we might need to see the [Create Fetch Event and Dispatch](https://w3c.github.io/ServiceWorker/#create-fetch-event-and-dispatch) response instead.